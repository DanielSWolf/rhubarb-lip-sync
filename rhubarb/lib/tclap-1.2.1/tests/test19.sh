#!/bin/sh

# success
../examples/test4 -BA --Bs asdf > tmp.out 2>&1

if cmp -s tmp.out $srcdir/test19.out; then
	exit 0
else
	exit 1
fi

