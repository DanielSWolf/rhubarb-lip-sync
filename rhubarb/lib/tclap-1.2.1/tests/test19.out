for string we got : asdf
for bool B we got : 1
for bool A we got : 1
